charts/newrelic-pixie/values.yaml

```diff
@@ -0,0 +1,28 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+# cluster: ""
+
+# The New Relic license key
+# licenseKey: ""
+
+# The Pixie API key
+# apiKey: ""
+
+# If you already have a secret where the New Relic license key and the Pixie API key are used, indicate its name here
+# customSecretName:
+# The key in the custom secret that contains the New Relic license key
+# customSecretLicenseKey:
+# The key in the custom secret that contains the Pixie API key
+# customSecretApiKeyKey:
+
+image:
+  repository: newrelic/newrelic-pixie-integration
+  tag: latest
+  pullPolicy: Always
```

Comment:
I'd wait to merge this until we have a production ready image with a versioned tag, even if it's beta.